03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,45 @@
+            <a href="http://kxan.com/category/news/local/">Local</a>
+            <a href="http://www.bloomberg.com/live">Economy</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+                <h2>Pokemon Gooooo!</h2>
+                <h5>Pokémon Go tops $160M in global revenue, time spent in-app remains high</h5>
+                <h6>Posted 1 hour ago by Darrell Etherington (@etherington)</h6>
+                <img src="http://assets.pokemon.com/assets/cms2/img/video-games/video-games/pokemon_go/169.jpg">
+                <p>I’m still trying to deal with losing Pokévision, too, but Niantic’s recent changes to the game don’t seem to be putting a damper on player appetite just yet — new data from analytics firm Sensor Tower reveals that users are still spending, on average, around 26 minutes per day in the app in the U.S., which is actually a slight increase compared to the average play duration the week prior to the update’s arrival.</p>
+                <p>And yes, users are still opening the app about the same amount of time, too — around six times per day, for the most part.</p>
+                <p>Sensor Tower’s sentiment analysis on the latest update also shows what I’ve heard anecdotally: People generally aren’t thrilled with the changes. But remember that one of the surest signs you’ve done something right in apps with significant uptake is that any changes are greeted with massive, primarily negative outcry.</p>
+            </article>
+            <img src="https://s.aolcdn.com/dims-global/dims3/GLOB/resize/1200x239/quality/80/https://tctechcrunch2011.files.wordpress.com/2016/08/pokemon-go-160-million-cumulative-revenue.png">
+        </section>
+        <aside>
+            <img src="https://media.giphy.com/media/O7EvAQv6gAxDG/giphy.gif">
+        </aside>
+    </main>
+    <footer>
+    <p style="font-size:10px">EWEN ROBERTS/FLICKR UNDER A CC BY 2.0 LICENSE</p>
+    <a href="https://www.facebook.com/"><img src="https://sunypoly.edu/home5/cmsfiles/favicon.ico/fb-logo.png" alt="facebook link" height ="20px" width="20px"/></a>
+    <a href="https://twitter.com/?lang=en"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/9/9f/Twitter_bird_logo_2012.svg/1259px-Twitter_bird_logo_2012.svg.png" alt="twitter link" height ="20px" width="20px"/></a>
+    <a href="https://www.instagram.com/accounts/login/"><img src="https://lh3.googleusercontent.com/aYbdIM1abwyVSUZLDKoE0CDZGRhlkpsaPOg9tNnBktUQYsXflwknnOn2Ge1Yr7rImGk=w300" alt="instagram link" height ="20px" width="20px"/></a>
+    <a href="https://www.linkedin.com/"><img src="http://theappshowbox.com/wp-content/uploads/2015/10/LinkedIn-Logo.png" alt="Linkedin link" height ="20px" width="20px"/></a>
+    <p>© 2013-2016 AOL Inc. All rights reserved. Aol Tech Privacy Policy About Our Ads Anti Harassment Policy Terms of Service Powered by WordPress.com VIP Fonts by Superman!</p>
+    </footer>
+</body>
+
+</html>
```

Comment:
Good use of h6 tags to emulate bylines. The animated gif is also a nice touch! Great work.